src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -59,8 +60,15 @@ public void setNeutralMode() {
     rightCenteringMotor.set(0);
   }
 
+  private double getRollerVelocity() {
+    return rollerMotor.get();
+  }
+
   @Override
   public void periodic() {
+
     // This method will be called once per scheduler run
+    SmartDashboard.putNumber("Intake/roller/Velocity RPS", getRollerVelocity());
```

Comment:
.get() is giving us a percent of maximum output which is different than velocity

Reply:
nuh uh